client/src/app.js

```diff
@@ -123,7 +126,9 @@ export function checkStreamResult(result) {
         };
 
         rtcConn.onconnectionstatechange = function() {
-            if(rtcConn.connectionState === 'connected') { updateState({ established: true }); }
+            if(rtcConn.connectionState === 'connected') { 
```

Comment:
Consider an enum or CONSTANT_STRINGS instead of hardcoded strings here.

Reply:
Please include the disconnect and failed cases here too.

Reply:
Not sure what you mean, there is "disconnected" and "failed" case in the function. As for Constant strings, I'll fix that right away